project/index.html

```diff
@@ -8,6 +8,11 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/3.0.3/normalize.min.css">
     <link rel="stylesheet" href="/public/styles.css" type="text/css">
 
+    <script>
+      var appParam = window.location.search.split("=")[1];
```

Comment:
Maybe use the [urlSearchParams spec](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams): 

```
var queryParams = new URLSearchParams(window.location.search);
var appParam = queryParams.get('app');
```

Reply:
I wanted to, but a large chunk of editorial machines don't run recent enough browsers http://caniuse.com/#search=URLSearchParams